src/PublishCodeCovCoverageTask/utils/webUtils.ts

```
@@ -17,241 +17,260 @@ import { Stream } from 'node:stream';
+                lastProgressTime = now;
+                lastReportedPercent = percent;
 
-                                options.onProgress!({
-                                    bytesReceived,
-                                    totalBytes: isNaN(totalBytes) || totalBytes <= 0 ? null : totalBytes,
-                                    percent
-                                });
-                            }
-                        });
-                    }
+                options.onProgress!({
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)